File: app/src/main/java/io/lerk/lrkFM/exceptions/EmptyDirectoryException.java
package io.lerk.lrkFM.exceptions;

import java.io.Serializable;

/**
 * {@link EmptyDirectoryException}.
 *
 * @author Lukas Fülling (lukas@k40s.net)
 */
public class EmptyDirectoryException extends Exception implements Serializable {
    static final long serialVersionUID = 10L;

    public EmptyDirectoryException(String message) {
        super(message);
    }
}

File: app/src/main/java/io/lerk/lrkFM/exceptions/EmptyDirectoryException.kt
package io.lerk.lrkFM.exceptions

import java.io.Serializable

/**
 * [EmptyDirectoryException].
 *
 * @author Lukas Fülling (lukas@k40s.net)
 */
class EmptyDirectoryException(message: String?) : Exception(message), Serializable {
    companion object {
        const val serialVersionUID = 10L
    }
}
